gfx/drivers_context/drm_ctx.c

```diff
@@ -563,7 +563,8 @@ static bool gfx_ctx_drm_egl_set_video_mode(gfx_ctx_drm_data_t *drm)
       case GFX_CTX_OPENGL_ES_API:
       case GFX_CTX_OPENVG_API:
 #ifdef HAVE_EGL
-         if (!egl_init_context(&drm->egl, (EGLNativeDisplayType)g_gbm_dev, &major,
+         if (!egl_init_context(&drm->egl, EGL_PLATFORM_GBM_KHR,
```

Comment:
The same sort of #ifdef should work for that as well -- the EGL header file has a macro for every EGL extension that it defines.

I can put together a pull request when I get home this afternoon.